readme.md

```diff
@@ -91,32 +94,39 @@ The second part is done in Bitbucket Server and involves creating an Application
 <img src="images/linkapplications.png" width="600"> <br/>
 
 To create the Application Link: 
-1. In Bitbucket Server go to **Administration** > **Application Links**. 
+1. In Bitbucket Server, go to **Administration** > **Application Links**. 
 2. Enter the application URL (see Application Link details page) and select **Create new link**. 
 3. Select **Continue** on the warning message. This is not a problem. 
-4. Complete all the fields in the form (see Application Link details page for some fields). 
-5. Leave **Create incoming link** unchecked. 
-6. Select **Continue**. 
+4. Complete the form: 
+   - Application Name - Enter a name for your Jenkins instance. 
+   - Application Type - Leave as Generic Application.
+   - Service Provider Name - Enter ??? Or same as description??? 
```

Comment:
@mhenschke-atl, do you know what the 'Service provider name' is used for? And how it differs from 'Application name'. I'm trying to add a bit of guidance here to reduce ambiguity.

Reply:
Not really :P From my testing it doesn't impact setup at all, and doesn't even seem to show up in the applink view. I assume it's a piece of additional identification in case the application name is too ambiguous, so it is at the user's discretion how they want to use it.
cc @dkjellin @khughes-atlassian in case they know for sure